engine/src/mac-window.mm

```diff
@@ -2051,7 +2051,10 @@ - (void)setFrameSize: (NSSize)size
 		if ([m_window_handle parentWindow] != nil)
 			[[m_window_handle parentWindow] removeChildWindow: m_window_handle];
 	
-		[m_window_handle orderOut: nil];
+		// CW-2015-09-21: [[ Bug 15979 ]] Call close instead of orderOut here to properly close
+		// windows that have been iconified.
+		[m_window_handle setReleasedWhenClosed: NO];
```

Comment:
This isn't necessary - setReleasedWhenClosed: is called in Realize when the window is created.